core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -109,7 +109,7 @@ var TopicEditorPage = function() {
     by.css('.protractor-test-topic-meta-tag-content-field'));
   var topicMetaTagContentLabel = element(
     by.css('.protractor-test-topic-meta-tag-content-label'));
-  var topicPageTitleFragmentField = element(
+  var topicPageTitleFragmField = element(
```

Comment:
Why is this changed?

Reply:
This was done to address krishnarao22's comment([here](https://github.com/oppia/oppia/pull/15423#discussion_r869575773)). It was done to shorten the variable name so that the line (line 53, core/tests/puppeteer/lighthouse_setup.js) was less than 80 characters long.

Reply:
Let's use "fragment" where we can.

Reply:
Done!